src/main/java/uk/gov/hmcts/divorce/solicitor/event/Applicant1SolicitorViewApplicant2ContactDetails.java

```diff
@@ -47,7 +46,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .page("applicant2ContactDetails")
             .pageLabel("Respondent Contact Details")
             .complex(CaseData::getApplicant2)
-                .readonly(Applicant::getAddress)
+                .readonly(Applicant::getNonConfidentialAddress)
                 .readonly(Applicant::getPhoneNumber)
```

Comment:
probably stupid question but we only seem to talk about confidential address  - we don't worry about them having phone number and email?

Reply:
Thought I'd put this in to try and close down the security vuln, then extend to other details later if it works well in prod. I'm slightly paranoid about changing these permissions in case something unexpected breaks and felt doing address on its own first could reduce risk